thetatauCMT/events/models.py

```diff
@@ -11,6 +13,12 @@
 from chapters.models import Chapter
 
 
+def get_event_upload_event(instance, filename):
```

Comment:
This function doesn't seem to be used and is unnecessary.
Along with the "from django.conf import settings" I think is also unnecessary at the top of the file, please remove them.